jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -151,4 +148,20 @@ private void logout() throws IOException, InterruptedException {
       new Command("docker", "logout", "localhost:" + port).run();
     }
   }
+
+  private void waitUntilReady() throws InterruptedException, MalformedURLException {
+    URL queryUrl = new URL("http://localhost:" + port + "/v2/_catalog");
+
+    for (int i = 0; i < 20; i++) {
+      try {
+        HttpURLConnection connection = (HttpURLConnection) queryUrl.openConnection();
+        int code = connection.getResponseCode();
+        if (code == HttpURLConnection.HTTP_OK || code == HttpURLConnection.HTTP_UNAUTHORIZED) {
+          return;
+        }
+      } catch (IOException ex) {
+        Thread.sleep(1000);
```

Comment:
Might be worth halving or quartering this: it's unlikely to need more than 250ms or 500ms.  Could use a `Stopwatch` and output how much time was waited?

No sleeping on a non-200/401?